CHANGELOG.md

```diff
@@ -1,3 +1,82 @@
+## Release 0.11.0
+
+New features:
+
+- UI Terminal to attach to Docker containers and see their output or `EXEC` a shell on them
+	[#650](https://github.com/weaveworks/scope/pull/650)
+  [#735](https://github.com/weaveworks/scope/pull/735)
+  [#726](https://github.com/weaveworks/scope/pull/726)
+- Added `scope version` command
+	[#750](https://github.com/weaveworks/scope/pull/750)
+- Various CPU usage gains for probe
+	[#742](https://github.com/weaveworks/scope/pull/742)
+	[#741](https://github.com/weaveworks/scope/pull/741)
+	[#737](https://github.com/weaveworks/scope/pull/737)
+- Show hostname of app you are connected to
+	[#707](https://github.com/weaveworks/scope/pull/707)
+- Add host memory and CPU usage metrics
+	[#711](https://github.com/weaveworks/scope/pull/711)
+- Add json support to app POST /api/report
+	[#722](https://github.com/weaveworks/scope/pull/722)
+- Add basic security warning.
+	[#703](https://github.com/weaveworks/scope/pull/703)
+- Update the docker version we embed into the scope image to 1.6.2 in sync with weave 1.3 changes.
+	[#702](https://github.com/weaveworks/scope/pull/702)
+- Show a spinner while node details are loading
+	[#691](https://github.com/weaveworks/scope/pull/691)
+- Procedural colors for nodes
+	[#694](https://github.com/weaveworks/scope/pull/694)
+- Mitigate one-line-of-nodes layouts
+	[#679](https://github.com/weaveworks/scope/pull/679)
+- Add basic kubernetes how-to to the readme
+	[#669](https://github.com/weaveworks/scope/pull/669)
+
+Bug fixes
+
+- Document debug options.
+	[#723](https://github.com/weaveworks/scope/pull/723)
+- Filter nodes that are only connected to themselves.
+	[#706](https://github.com/weaveworks/scope/pull/706)
+- Merge image metadata into container metadata so image labels can be used to hide containers.
+	[#705](https://github.com/weaveworks/scope/pull/705)
+- Dont send topology options when requesting node details
+	[#704](https://github.com/weaveworks/scope/pull/704)
+- Don't apply filters to node endpoints & update details panel logic
```

Comment:
Show details for stopped container in the default view, by not applying filters to the node details endpoints.